### PR TITLE
resolve conflicts of population size default configurations

### PR DESCRIPTION
**Pull request by michaella21**
just change some values in component and tests according to the change in  previous PR in vivarium

### CODE REVIEW

File: src/vivarium_public_health/population/base_population.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class BasePopulation:
             'age_start': 0,
             'age_end': 125,
             'exit_age': None,
-            'population_size': 10000,
+            'population_size': 100,
```
Comment (collijk) [inline src/vivarium_public_health/population/base_population.py]: This should still fail, I think.  `population_size` is set by the population manager in vivarium.

File: tests/disease/test_disease.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def test_dwell_time(assign_cause_mock, base_config, disease, base_data):
     base_config.update({
         'time': {'step_size': time_step},
         'population': {'population_size': 10}
-    }, **metadata(__file__))
+    }, layer='override')
```
Comment (collijk) [inline tests/disease/test_disease.py]: Why the change here?

File: src/vivarium_public_health/population/base_population.py (hunk 1 above)
Comment (michaella21) [reply src/vivarium_public_health/population/base_population.py]: when I ran the pytest before this change, test_base_population failed with the population size 100 set by population manager that was still a component level configs, then base_population tries to override at the same config level.

File: tests/disease/test_disease.py (hunk 2 above)
Comment (michaella21) [reply tests/disease/test_disease.py]: probably i didn't think enough :(

**Comment by halfhorst** [comment]
So this just eliminates a setting of population_size because it would break the "multiple components shouldn't set defaults for the same thing" PR ?
